packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django==1.2 
+django==1.4.11 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2015-5964   
**Check Name:** python-django: Denial-of-service possibility in logout() view by filling session store   
**Severity:** MEDIUM   
**Fixed Version:** 1.7.10, 1.4.22   
**Description:** The (1) contrib.sessions.backends.base.SessionBase.flush and (2) cache_db.SessionStore.flush functions in Django 1.7.x before 1.7.10, 1.4.x before 1.4.22, and possibly other versions create empty sessions in certain circumstances, which allows remote attackers to cause a denial of service (session store consumption) via unspecified vectors.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2015-5964